app/models/issue.rb

```diff
@@ -13,6 +13,10 @@ class Issue < ApplicationRecord
 
   ransack_alias :state, :aasm_state
 
+  ransacker :reason_code, formatter: proc { |v| IssueReason.find_by_code(v.to_sym).id } do |parent|
```

Comment:
`IssueReason.find_by_code` devuelve `nil` cuando no existe el código y acá se levantaría ``NoMethodError: undefined method `id' for nil:NilClass``.
Talvez es tan simple como `IssueReason.find_by_code(v)&.id`, pero habría que escribir un spec o dos.

Reply:
listo